e2e-tests/version-service/conf/operator.9.9.9.psmdb-operator.json

```diff
@@ -34,28 +82,58 @@
             "image_hash": "255df9d2157eac6d9f452780b918b352da90814af0ab3210ffbe23177e6d4cf8",
             "status": "available",
             "critical": false
+          },
+          "4.2.13-14": {
+            "image_path": "percona/percona-server-mongodb:4.2.13-14",
+            "image_hash": "5419687291e7fa1e3bf2ae1fa0340cbf72f38f8ad70c0800fa7b8c145f5e75ae",
+            "status": "available",
+            "critical": false
+          },
+          "4.2.12-13": {
+            "image_path": "percona/percona-server-mongodb:4.2.12-13",
+            "image_hash": "dda89e647ea5aa1266055ef465d66a139722d9e3f78a839a90a9f081b09ce26d",
+            "status": "available",
+            "critical": false
+          },
+          "4.2.11-12": {
+            "image_path": "percona/percona-server-mongodb:4.2.11-12",
+            "image_hash": "1909cb7a6ecea9bf0535b54aa86b9ae74ba2fa303c55cf4a1a54262fb0edbd3c",
+            "status": "available",
+            "critical": false
+          },
+          "4.2.8-8": {
+            "image_path": "percona/percona-server-mongodb:4.2.8-8",
+            "image_hash": "a66e889d3e986413e41083a9c887f33173da05a41c8bd107cf50eede4588a505",
+            "status": "available",
+            "critical": false
+          },
+          "4.2.7-7": {
+            "image_path": "percona/percona-server-mongodb:4.2.7-7",
+            "image_hash": "1d8a0859b48a3e9cadf9ad7308ec5aa4b278a64ca32ff5d887156b1b46146b13",
+            "status": "available",
+            "critical": false
           }
         },
         "pmm": {
-          "2.24.0": {
-            "image_path": "percona/pmm-client:2.24.0",
-            "image_hash": "bf0cdfd9f9971964cb720a92e99da1a75367cf6a07deec9367ca6b80e78b0f89",
+          "2.27.0": {
```

Comment:
so old?

Reply:
good to see you. I plan to fix it after release changes when we will know all our new images.